ripe/atlas/tools/probes/__init__.py

```diff
@@ -44,7 +44,7 @@ def get_many(cls, ids):
                 fetch_ids.append(str(pk))
 
         if fetch_ids:
-            kwargs = {"id__in": ",".join(fetch_ids)}
```

Comment:
Wouldn't it be better to make Cousteau smart enough to know to do this to lists of integers rather than forcing this format outside of the library?